### ISSUE TITLE
learn odoo

### ISSUE THREAD

**Issue by spectra10008** [odoo/odoo#22805]
What I need to learn before starting odoo

**Comment by jpaulin** [comment]
This blog has taught me a lot: https://www.odoo.yenthevg.com/ 

Learn 
- set up (=install)
- how to run and stop the service
- what user roles are (especially how the admin user can do things, compared to other users)
- basics of products, and most likely also from Tasks, Timesheet, perhaps Website, Employees, Invoicing

Also I benefited a lot from a real, printed book for Odoo. I work as implementor / admin. 

Since you are running a specific version of Odoo, like Odoo 9, 10, or 11, I really recommend then going exactly for instructions that are aimed at that particular version. As you learn more, you can distinguish and sift through the information independently, and better apply knowledge from many sources (even though it's not exactly meant for the that version).

It depends, whether you need to be a implementor (doing code perhaps), or basically want to use the system through the web browser (UI) mostly.

**Comment by sswapnesh** [comment]
Hello @spectra10008 
This is not the right place for this type of questions. Please use odoo forums or Official document instead. ([https://www.odoo.com/forum/help-1](https://www.odoo.com/forum/help-1) and [https://www.odoo.com/page/docs](https://www.odoo.com/page/docs))

**Comment by jpaulin** [comment]
@sswapnesh @spectra10008 Yes, true. "Issues" in github is reserved to specific reporting of a glitch or similar thing in the software itself.

**Comment by Yenthe666** [comment]
Duplicate of https://github.com/odoo/odoo/issues/22801
Quoting myself from there:
> P.S: You can find the forums at help.odoo.com or you can have a look in the documentation at https://www.odoo.com/page/docs

Please don't create duplicate topics.

**Comment by spectra10008** [comment]
Oopss.. sorry !!
